### PR TITLE
project link & getting project name from crowdin

### PR DESCRIPTION
**Pull request by Pacmard**
The bot sends a link to the changed project at the end of the message

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ async function handleRequest(request: Request) {
     }
 
   const body = JSON.stringify({
-    content: `New changes were made in ${projectName}!\n${updates.join('\n')}`,
+    content: `New changes were made in ${projectName}!\n${updates.join('\n')} \nLink: https://crwd.in/${res.events[0].project}`,
```
Comment (velddev) [inline src/index.ts:53]: Maybe we can do something like this?

```suggestion
content: [
  `New changes were made in ${projectName!}`,
  ...updates,
  `Link to project: ${res.events[0].project}`,
].json("\n"),
```

Comment (Pacmard) [reply src/index.ts]: content is not used anymore, message being sent as embed

**Review by Xetera — APPROVED** [review]
lgtm

File: src/index.ts [hunk 2]
```diff
@@ -40,19 +38,12 @@ async function handleRequest(request: Request) {
...
-    }
-
+  let projectName = res.events[0].project
+  
   const body = JSON.stringify({
     embeds: [{
       "title": `New changes were made in ${projectName}!`,
-      "description": `${updates.join('\n')}\n\n[Project Link](https://crwd.in/${res.events[0].project})`
+      "description": `${updates.join('\n')}\n\n[Project Link](https://crwd.in/${projectName})`
     }],
```
Comment (ShockTr) [inline src/index.ts]: I think it will look nicer if embed had color like green.

**Comment by bashcoders** [comment]
what if i start spam here also?

**Comment by bashcoders** [comment]
> what if i start spam here also?

hehe

**Comment by bashcoders** [comment]
https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494

**Comment by bashcoders** [comment]
https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494

**Comment by bashcoders** [comment]
https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494

**Comment by bashcoders** [comment]
https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494

**Comment by bashcoders** [comment]
https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494https://discord.com/channels/@me/816581392623337494

**Pull request merged**